server.js

```diff
@@ -1,30 +1,126 @@
 import express from 'express'
-import bodyParser from 'body-parser'
 import cors from 'cors'
+import listEndpoints from 'express-list-endpoints'
+
+import netflixTitles from './data/netflix-titles.json'
+
+const movies = netflixTitles.filter((titel) => titel.type === "Movie")
+const tvShows = netflixTitles.filter((titel) => titel.type === "TV Show")
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
 const port = process.env.PORT || 8080
 const app = express()
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors())
-app.use(bodyParser.json())
+app.use(express.json())
 
-// Start defining your routes here
+// Listing all endpoints
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send(listEndpoints(app))
+})
+
+// Listing all titles, no queries
+app.get('/titels', (req, res) => {
+  res.json({ titels: netflixTitles })
+})
+
+// Listing titles by id
+app.get('/titels/:id', (req, res) => {
+  const { id } = req.params
+  const foundTitel = netflixTitles.find((titel) => titel.show_id === +id)
+  if (id) {
+    res.status(200).json({ show: foundTitel })
+  } else {
+    res.status(404).json({ status_message: "Not found" })
+  }
+})
+
+app.get('/movies', (req, res) => {
+  const { year } = req.query
+  const { yearAfter } = req.query
+  const { yearBefore } = req.query
+  const { cast } = req.query
+  const { director } = req.query
+  const { country } = req.query
```

Comment:
`const { year, yearAfter, yearBefore, cast, director, country } = req.query` let it DRY 🤓